### ISSUE TITLE
demo plan/things we need to do before we can demo

### ISSUE THREAD

**Issue by FawziyahAlebiosu** [MLH-Fellowship/0.5.2-cms#27]
@amy-lei @ebonnecab 
 - add license to our repo
 - video demo

questions for y'all: 
- do we want to create a presentation to go along with the video? Or do we want to just have points in our google docs that we will go over as we record the video?
- how do we want to record our video? Use loom? get on a zoom call and record it? any suggestions?

**Comment by amy-lei** [comment]
@FawziyahAlebiosu 
* Maybe a presentation? Or doc is kind of hard to follow unless we clean it up. 
* I would be down for zoom! I managed to fix it the other day. 

In terms of time though, tonight would be ideal. But realistically, we might even need tonight to really finish everything and ensure we have a working project?

**Comment by ebonnecab** [comment]
@FawziyahAlebiosu i'm also down to record a zoom call and i think we can have a presentation to go with the video so we can explain everything. I might be working into the night but I am down to record it super early. I can be up by like 5 PST or something

**Comment by amy-lei** [comment]
@ebonnecab  :') Same here! 99% sure ill be staying up working still.

**Comment by FawziyahAlebiosu** [comment]
Yeah same! Okay plan of action:

1. I’m going to finish the cli to include the nested commands and editing by tonight. I’ll also record a gif/screen recording of how it works. 
 However I need a decision from both of you in how to approach links.
Are we going to put a dummy link/placeholder for right now OR are we going to take Raymond’s suggestion and deploy it? 

2. So it looks like the consensus is that we’re doing a presentation! I’ll start one in like 3 hours and share it with you all

3. I see ebonne is fine with recording a zoom/meet call, I’m down for that! Amy, are you okay with that? If you are, someone suggest a time and we can all see if that works for us. 

We’re in the final stretch y’all :)))

**Comment by ebonnecab** [comment]
I can try to deploy it tonight once its done. Sometimes that can be a challenge due to weird stuff like packages and environments, but worse comes to worst we can use a placeholder link

**Comment by amy-lei** [comment]
@FawziyahAlebiosu  Does linking it to the local server `http://127.0.0.1:5000/` not work? But yeah, I agree with Ebonne.

**Comment by FawziyahAlebiosu** [comment]
It ~kinda~ does. I needed to run it on the command line first, then ctrl c to quit because I couldn’t type python connections.py while the server was running. 
What if I added to the readme the following right after “navigate to 127....”:
Run the connections.py file with the command you want AFTER deploying the application locally.
That way, the user can infer that the python file will not work as it should unless they run it in their computer first?
And that could be a possible solution/placeholder ?

**Comment by FawziyahAlebiosu** [comment]
> I can try to deploy it tonight once its done. Sometimes that can be a challenge due to weird stuff like packages and environments, but worse comes to worst we can use a placeholder link

Yup! I think I will do that with the server link!
